packetizer.go

```diff
@@ -41,6 +43,24 @@ func NewPacketizer(mtu int, pt uint8, ssrc uint32, payloader Payloader, sequence
 	}
 }
 
+func (p *packetizer) EnableAbsSendTime(value int) {
+	p.AbsSendTime = value
+}
+
+func toNtpTime(t time.Time) uint64 {
+	var s uint64
+	var f uint64
+	u := uint64(t.UnixNano())
+	s = u / 1e9
+	s += 0x83AA7E80 //offset in seconds between unix epoch and ntp epoch
```

Comment:
does this deal with the 2038 problem?

Reply:
t.UnixNano will roll over in 2554. the NTP format will roll over sooner (it rolls over every 136 years). 
The timestamp we're deriving from it (abs-send-time) rolls over every 64 seconds -- i don't think it's a problem.